build/lib/release.sh

```diff
@@ -374,10 +374,23 @@ function kube::release::create_docker_images_for_server() {
         ln "${binary_file_path}" "${docker_build_path}/${binary_name}"
         ln "${KUBE_ROOT}/build/nsswitch.conf" "${docker_build_path}/nsswitch.conf"
         chmod 0644 "${docker_build_path}/nsswitch.conf"
-        cat <<EOF > "${docker_file_path}"
+        if [[ "${binary_name}" =~ kube-apiserver ]]; then
+          cat <<EOF > "${docker_file_path}"
+FROM debian:stretch
+RUN apt-get update && apt-get install -y -q --no-install-recommends --no-install-suggests --fix-missing gcc git libc6-dev make
+RUN cd /root && git clone git://git.kernel.org/pub/scm/linux/kernel/git/morgan/libcap.git
+RUN cd /root/libcap/progs && make
 FROM --platform=linux/${arch} ${base_image}
+COPY --from=0 /root/libcap/progs/setcap /bin/
 COPY ${binary_name} /usr/local/bin/${binary_name}
+RUN ["setcap", "cap_net_bind_service=+ep", "/usr/local/bin/${binary_name}"]
```

Comment:
baking per-binary logic into the release script like this is really questionable. if the control plane images can't be handled generically, maybe they should have their own dockerfiles rather than a generated one?

Reply:
+1, as an approver of build/
Also if these dockerfiles are non trivial they should be on disk and use ARG instead of being generated.

Reply:
AFAIK only kube-apiserver and kube-proxy to require non trivial Dockerfiles. Where would these Dockerfiles live if I were to add those?

Reply:
Added a dockerfile and relevant ARGs.